plugins/powerd/meson.build

```diff
@@ -1,6 +1,5 @@
-if get_option('plugin_powerd').disable_auto_if(host_machine.system() != 'linux').require(gio.version().version_compare('>=2.58'),
-     error_message: 'gio 2.58 or later needed for powerd').allowed()
-  cargs = ['-DG_LOG_DOMAIN="FuPluginPowerd"']
+if host_machine.system() == 'linux' and gio.version().version_compare('>=2.58')
```

Comment:
we can drop the gio version specific compare and just look for gio.  our base requirement for gio is now 2.72

Reply:
Actually, we can drop the `host_machine.system() == 'linux'` too -- there's nothing Linux specific there.